src/components/signature-area/signature-area.ts

```diff
@@ -26,6 +26,9 @@ export class SignatureAreaComponent implements ControlValueAccessor {
   public clearAction: string;
   public actionLess: boolean;
 
+  @Input()
+  oldSignature: string;
```

Comment:
You could use the `signature` property for keeping the old and new data.

Reply:
Fixed